package.json

```diff
@@ -34,7 +35,7 @@
     "build": "yarn clean && yarn compile",
     "prepare": "husky",
     "publish": "semantic-release",
-    "lint": "eslint . --ext .ts,.tsx",
+    "lint": "ESLINT_USE_FLAT_CONFIG=false eslint . --ext .ts,.tsx",
```

Comment:
temporary to support the transition, otherwise the current eslint config won't work since I upgraded to eslint 9 in this PR